src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -1216,23 +1213,31 @@ void HexagonGame::initLua()
     // ------------------------------------------------------------------------
     // Register Lua function to get random seed for the current attempt:
     addLuaFn("u_getAttemptRandomSeed", //
-        [this] { return rng.seed(); })
-        .doc(
-            "Obtain the current random seed, automatically generated at the "
-            "beginning of the level. `math.randomseed` is automatically "
-            "initialized with the result of this function at the beginning of "
-            "a level.");
+             [this]{ return rng.seed(); })
+            .doc(
+                    "Obtain the current random seed, automatically generated at the "
+                    "beginning of the level. `math.randomseed` is automatically "
+                    "initialized with the result of this function at the beginning of "
+                    "a level.");
```

Comment:
Be sure to stick to the ``.clang-format`` that Vee has defined for us.

Reply:
can you go away with it when the thing is god damn raw?